web/pages/[username]/[contractSlug].tsx

```diff
@@ -75,9 +88,10 @@ export async function getStaticPropz(props: {
   return {
     props: {
       contract,
-      bets,
+      bets: bets.slice(0, 100),
```

Comment:
Another thing is that that some parts of the client may assume it has all the bets. I think you may need to update, e.g. the contract leaderboards code, or the bets tab to not show an incorrect count.

These cases seem blocking to me.

Reply:
done!